contribs/gmf/src/search/component.js

```diff
@@ -226,10 +239,10 @@ gmf.SearchController = class {
     this.gmfTreeManager_ = gmfTreeManager;
 
     /**
-     * @type {gmf.FulltextSearchService}
+     * @type {gmf.search.fulltextSearch.fulltextSearch_}
```

Comment:
That is not good, you can not depend on a private type defined in a different file. See comment in the service file.